Orchestrator/README.md

```diff
@@ -39,26 +54,32 @@ Thus, use of [BF command line tool][7] to prepare and optimize the model for you
    * See `bf orchestrator:basemodel:list` for alternate models. You may need to experiment with the different models to find which performs best for your language domain.
 3. Combine the label file .lu from (1) with the base model from (2) to create a *snapshot* file with a .blu extension.
    * Use [`bf orchestrator:create`][16] to create just a single .blu snapshot file for all Lu/json/qna tsv files for dispatch scenario.
+-->
+
+<!--This is insufficient information, but again the other article covers this.
```

Comment:
I agree, now that we have the other doc, we can trim this section, and refer to the other doc.

Reply:
I'll leave it up to you to decide what's best here :)

Reply:
Yes, please add a prerequisites note to the other article. (I'm keeping a similar note in this article.) I'll call this good for now.